paper.md

```diff
@@ -332,34 +328,36 @@ To this end, we look forward to supporting a wider of collection of collaborator
 <!-- JOSS: Do the authors describe how this software compares to other commonly-used packages?-->
 
 
-There are many existing software platforms that provide support for the different functionalities that Empirical also provides.
-However, most are not in C++, and there is value in this functionality being easily available to programers .
+There are many existing software platforms that provide functionalities overlapping with Empirical.
+However, most are not in C++, and there is value in this functionality being easily available to programmers.
```

Comment:
C++ programmers (instead of just `programmers`)? Or is the more general term intentional?